.github/workflows/visual-testing.yml

```
@@ -22,11 +22,14 @@
         run: npm install -g pnpm
 
       - name: Install dependencies
-        run: make install
+        run: pnpm install
+
+      - name: Install playwright
+        run: pnpm exec playwright install  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)